libs/SalesforceSDKCore/SalesforceSDKCore/Classes/PushNotification/NotificationCategoryFactory.swift

```diff
@@ -0,0 +1,45 @@
+import Foundation
+
+@objc(SFSDKNotificationCategoryFactory)
+@objcMembers
+public class NotificationCategoryFactory: NSObject {
```

Comment:
Created a NotificationCategoryFactory class so it was easier to test the Category creation for the Notificaiton Types.  This was to ensure we test the JSON we get back from our calling to the Server Endpoint for Notificaitgon Types.